### ISSUE TITLE
Using the pretrained models cannot replicate the reported results.

### ISSUE THREAD

**Issue by chenguanqi** [devendrachaplot/Neural-SLAM#44]
Hi,

I evaluated the pretrained models in the validation set of the Gibson with 2 Nividia GTX2080Ti GPUs by the following instruction. 
```shell
python  main.py --split val_mt --eval 1 \
--auto_gpu_config 0 -n 14 --num_episodes 71 --num_processes_per_gpu 7 \
--map_size_cm 4800 --global_downscaling 4 \
--load_global pretrained_models/model_best.global --train_global 0 \
--load_local pretrained_models/model_best.local  --train_local 0 \
--load_slam pretrained_models/model_best.slam  --train_slam 0 \
--print_images 1 -d results/ --exp_name exp_pretrained
```

And I got the following outputs:
Final Exp Area: 
6.61225, 10.99293, 14.25592, 16.76154, 18.83926, 20.62707, 22.04239, 23.27103, 24.32681, 25.19376, 25.84527, 26.43538, 26.97672, 27.41279, 27.86179, 28.27807, 28.66676, 29.05918, 29.41082, 29.70030, 29.96494, 30.23517, 30.49235, 30.70271, 30.93785, 31.15789, 31.34198, 31.51161, 31.66111, 31.81541, 31.96644, 32.09205, 32.19407, 32.27754, 32.37009, 32.46788, 32.56964, 32.65248, 32.71281, 32.77295,
Final Exp Ratio: 
0.22896, 0.38165, 0.49123, 0.57334, 0.63864, 0.69213, 0.73176, 0.76435, 0.79112, 0.81191, 0.82767, 0.84121, 0.85233, 0.86099, 0.86996, 0.87746, 0.88441, 0.89109, 0.89688, 0.90177, 0.90632, 0.91083, 0.91484, 0.91813, 0.92164, 0.92492, 0.92760, 0.93034, 0.93274, 0.93522, 0.93779, 0.93969, 0.94120, 0.94239, 0.94376, 0.94523, 0.94671, 0.94795, 0.94882, 0.94975,

The mean value of the relative coverage is 83.7%, which is much lower than the reported result (94.8%).
How can I replicate the reported result?

**Comment by devendrachaplot** [comment]
The code is printing the average intermediate explored area every 25 steps, the final explored ratio is 0.94975 or 94.975%.
